README.md

```diff
@@ -1,15 +1,73 @@
-This is the Sublime Text client for [TabNine](https://tabnine.com), the all-language autocompleter.
+## **TabNine For Sublime**
 
-- Indexes your whole project, reading your .gitignore to determine which files to index.
-- Type long variable names in just a few keystrokes using the mnemonic completion engine.
+
+### **TabNine Overview:**
+___
+
+This is the Sublime Text client for [TabNine](https://tabnine.com), advanced AI based autocomplete for all programming languages. TabNine Indexes your entire project by reading your `.gitignore` [or others](https://www.tabnine.com/faq#nongit), and determines which files to index. 
+
+- Enter long variable names using  minimal keystrokes and leverage a mnemonic completion engine.
 - Zero configuration. TabNine works out of the box.
-- Highly responsive: typically produces a list of suggestions in less than 10 milliseconds.
+- Highly responsive: Produces a list of suggestions in less than 10 milliseconds on average.
+
+TabNine is part of [Codota](https://www.codota.com/) 
+
+<img src="https://raw.githubusercontent.com/codota/tabnine-vscode/master/assets/tabnine.gif" height="300" style="border-radius: 4px" />
```

Comment:
change url?

Reply:
I've put this url specific to point to the vscode image. Would you prefer that I copy the gif to this repo?